openshift/4.0/templates/jenkins/jenkins.yaml

```diff
@@ -0,0 +1,139 @@
+kind: Template
```

Comment:
Adding jenkins deployment config to repo to make it easy to change the resources.